playground/fibonacci_using_for.js

```diff
@@ -0,0 +1,26 @@
+function fibo (idx) {
+    let sumNum = 0;
+    
+        if (idx == 1 || idx == 2) {
+            return 1
+        } else if( idx >= 3) {
+            let curNum=1 ;
+            let bfNum=1 ;
```

Comment:
불필요한 스페이스가 들어간 것 같네요.
```suggestion
            let curNum=1;
            let bfNum=1;
```